### DISCUSSION TITLE
Are there any plans to port to Linux?

### DISCUSSION THREAD

**Discussion by allblue** [1Remote/1Remote#488] [General]
Alternative-credential, MySQL database and Launcher features are too powerful, eager to use it on linux as well.

**Comment by majkinetor** [comment]
I would also like this to happen, but I wouldn't count on that soon, unless somebody chip in to offer help. Custom protocols/apps can be used to provide similar connection functionality cross-platform (using for example things like Free RDP), all with  features you mention naturally being available and more, for example launching a group via tag which is awesome feature. 

Good thing is that code is cross-platform dot net 6 already, so that will help a lot if such effort is undertaken.

**Reply by allblue** [reply]
Good news, compared to Windows, Linux lacks this kind of integrated remote management tool, whether it is the paid RemoteDestopManager, finalssh, or the open source remmina, windterm, woterm, tabby and so on, they have a lot of shortcomings in terms of protocol richness and usability. I am very looking forward to 1remote’s early porting success.

**Reply by VShawn** [reply]
Bad news, I'm not a Linux desktop user, I'm not familiar with Linux desktop. so I don't have any plans to port to Linux at the moment.
